### DISCUSSION TITLE
Zを押すとZとVが反応するようになった

### DISCUSSION THREAD

**Discussion by akki-F** [Yowkees/keyball#415] [Q&A] [answered]
keyball61を普段づかいしていたのですが、突然ZをおすとVもあわせて反応するようになってしまいました。
どの部分を見直すまたは半田しなおすとよいでしょうか。
よろしくお願いします。
![IMG_6614](https://github.com/Yowkees/keyball/assets/45144201/06e773bd-be18-4241-8052-3fafe6a0ae62)

**Comment by Yowkees** [comment, marked as answer]
ご連絡ありがとうございます。

Keyball61はProMicroのピン数制限の関係でDuplexマトリクスを採用しており、ピンを共用しているZとVで問題が起きているようですね。
下記画像の丸で示した３つのダイオードがZとVに関連するものですので、これらの半田付けを見直していただきたいのと、できれば新しいダイオードへ交換してみてください。

以上、お手数ですがご確認お願いいたします。

![KB61_duplexfix](https://github.com/Yowkees/keyball/assets/84209772/d50c7254-5221-4bc5-bf3d-c00f04470b6f)

**Comment by akki-F** [comment]
ご連絡ありがとうございます。
ご指摘の場所をはんだしなおしたところ問題なく動作するようになりました。
当チケットはcloseします。
